src/main.cpp

```diff
@@ -329,16 +331,16 @@ int main(int argc, char **argv)
 {
     parse_options(argc, argv);
 
-    bool gui_error = false;
-
     if (g_options.gui)
     {
 #ifdef WIN32
         unassemblize::gui::ImGuiWin32 gui;
         unassemblize::gui::ImGuiStatus status = gui.run(g_options);
         return int(status);
 #else
-        gui_error = true;
+        unassemblize::gui::ImGuiGLFW gui;
```

Comment:
This works on all platforms?

Reply:
OpenGL + GLFW in theory works on windows, mac, and linux. I haven't tested in on my windows machine yet though. Also, I do like having an else gui_error = true block as the default if someone is using something other than windows mac or linux - so I added more specific ifs, and restored the else error handling from before.